### PR TITLE
Decrease peer reputation on bad transactions

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: core/service/src/lib.rs [hunk 1]
```diff
@@ -629,10 +636,13 @@ where
 				let import_future = import_future
 					.then(move |import_result| {
 						match import_result {
-							Ok(_) => report_handle.report_peer(who, reputation_change),
+							Ok(_) => report_handle.report_peer(who, reputation_change_good),
 							Err(e) => match e.into_pool_error() {
 								Ok(txpool::error::Error::AlreadyImported(_)) => (),
-								Ok(e) => debug!("Error adding transaction to the pool: {:?}", e),
+								Ok(e) => {
```
Comment (marcio-diaz) [inline core/service/src/lib.rs]: Is there a problem when decreasing reputation of already imported? (why did you change it?)

Comment (arkpar) [reply core/service/src/lib.rs]: Transactions are propagated by push. We can't know in advance if the peer we're sending to already has it. So we don't punish or reward for duplicates.
